backend/src/models/animal.ts

```diff
@@ -32,8 +32,7 @@ export interface AnimalAttributes {
 }
 
 // Some attributes are optional in `Animal.build` and `Animal.create` calls
-// eslint-disable-next-line 
-export interface AnimalCreationAttributes extends Optional<AnimalAttributes, "id"> { }
+export type AnimalCreationAttributes = Optional<AnimalAttributes, "id">
```

Comment:
I believe typescript requires this to be an interface in order to extend the class? Or is it possible to do it like this now?